Update/UpdateDriversAndUEFI.md

```diff
@@ -25,24 +55,35 @@ adb shell "/sdcard/msc.sh"
 
 ## Updating Drivers
 
-- Extract the drivers, Extract driver updater, and from the command prompt in the DriverUpdater.exe directory:
+**_⚠️ WARNING: From now on we will assume X: is the Win partition for all commands. Replace them correctly with what you previously picked or you will lose data on your PC._**
+
+- Extract the driver package, extract the driver updater zip, and open a command prompt in the directory containing DriverUpdater.exe:
 
 ```batch
 DriverUpdater.exe -d "<path to extracted drivers>\definitions\Desktop\ARM64\Internal\epsilon.txt" -r "<path to extracted drivers>" -p X:\
 ```
 
-- Once it is done, you can reboot your phone using ```adb reboot bootloader```. You will be able to boot to Android™ and your phone will work normally. Set it up if you need it.
+- Once it is done, you can reboot your phone using:
+
+```batch
+adb reboot bootloader
+```
 
 You will be back into Surface Duo's bootloader. 
 
 ## Updating UEFI
 
-- Simply use the latest UEFI image
+Simply use the latest UEFI image
 
-Reboot your device to fastboot, using adb or from the recovery.
-      
-Let's boot the custom UEFI, from a command prompt:
+- Assuming you are in Surface Duo´s bootloader, run the following command to boot into Windows:
```

Comment:
For some reason your "'" characters are not the right ones, this may cause some issues